pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -263,7 +285,7 @@ def is_optimized_sync(repository, remote, url):
 
     """
     with tempfile.TemporaryDirectory("."):
-        result = get_repomd_file(remote, url)
+        result = get_repomd_file(remote, url, raise_err=False)
```

Comment:
For what it's worth, I'm not sure if this case and particularly the one below should *actually* ignore errors.  In that case maybe we should just ditch `raise_err` and make it always throw exceptions, and just handle them as needed.

Reply:
We're looking  for the possible existence of a repomd.xml file - if it's "positively not here", we just move on, because we're the only ones who care. (I think)